### PR TITLE
doc: Revise packages filtering doc section

### PR DESCRIPTION
**Pull request by m-blaha**
Resolves: https://github.com/rpm-software-management/dnf5/issues/1354

### CODE REVIEW

File: doc/dnf5.conf.5.rst [hunk 1]
```diff
@@ -741,6 +739,12 @@ configuration.
 
     Default: ``75``
 
+.. _disable_excludes_options-label:
+``disable_excludes``
+    :ref:`list <list-label>`
+
+    Used to disable packages filtering. It can include a list of repository IDs (globs allowed) for which repository-specific excludes are ignored. If it contains the special value ``main`` excludes from the main configuration files are bypassed. If it contains the special value ``*`` no excludes filtering is applied at all.
```
Comment (ppisar) [inline doc/dnf5.conf.5.rst:746]: Please add commas after the conditional sentences: "value ``main``, excludes" and "value ``*``, no excludes".

File: doc/misc/filtering.7.rst [hunk 2]
```diff
@@ -28,42 +25,49 @@
...
+===========
+
+Additionally, the versionlock functionality is implemented using excludes filtering. However, these excludes are applied only during transactional operations.
+For details see :manpage:`dnf5-versionlock(8)`, :ref:`Versionlock command <versionlock_command_ref-label>`.
+
+
+Modular filtering
 =================
 
 Please see `the modularity documentation` for details on how Modular Filtering works.
```
Comment (ppisar) [inline doc/misc/filtering.7.rst:64]: I cannot find `the modularity documentation`. If the documentation does not exist, consider removing this sentence.

**Comment by ppisar** [comment]
Otherwise the patch is good.

File: doc/misc/filtering.7.rst (hunk 2 above)
Comment (m-blaha) [reply doc/misc/filtering.7.rst:64]: Oh, right. The documentation exists, so let me revise it quickly and add to the package.

File: doc/misc/filtering.7.rst [hunk 3]
```diff
@@ -28,42 +25,49 @@
...
+===========
+
+Additionally, the versionlock functionality is implemented using excludes filtering. However, these excludes are applied only during transactional operations.
+For details see :manpage:`dnf5-versionlock(8)`, :ref:`Versionlock command <versionlock_command_ref-label>`.
+
+
+Modular filtering
+=================
+
+Please see :ref:`the modularity documentation <modularity_misc_ref-label>` for details on how Modular Filtering works.
```
Comment (ppisar) [inline doc/misc/filtering.7.rst:64]: Now, when modularity(7) page is packaged, shouldn't this link mention both the manual page and the modularity_misc_ref-label. Similar to one in Versionlock section above?

Comment (m-blaha) [reply doc/misc/filtering.7.rst]: Yeah, we should reference other docs in uniform way. Fixed.

**Review by ppisar — APPROVED** [review]
Good. The build failure is unrelated (#1625).

**Pull request merged**